src/commands/generate/project.ts

```diff
@@ -0,0 +1,246 @@
+import { Flags } from '@oclif/core'
+import { execSync } from 'child_process'
+import { compile } from 'ejs'
+import { access, mkdir, readFile, rm, stat, writeFile } from 'fs/promises'
+import { prompt } from 'inquirer'
+import { basename, dirname, join, resolve } from 'path'
+import { Command } from '../../utils/command/command'
+import { computePackageJSON } from '../../utils/package-json'
+
+export default class Project extends Command {
+  static description = 'Generate fastify project'
+
+  static args = [
+    { name: 'name', required: false, description: 'Name of the project.' }
+  ]
+
+  static flags = {
+    location: Flags.string({ description: 'Location to place the project.' }),
+    overwrite: Flags.boolean({ description: 'Force to overwrite the project location when it exist.', default: false }),
+    language: Flags.string({ description: 'Programming Language you would like to use in this project.' }),
+    lint: Flags.string({ description: 'Lint Tools you would like to use in this project.' }),
+    test: Flags.string({ description: 'Test Framework you would like to use in this project.' }),
+    help: Flags.help()
+  }
+
+  shouldOverwrite = false
+
+  async run (): Promise<void> {
+    const { args, flags } = await this.parse(Project)
+
+    // validate
+    if (flags.language !== undefined) this.corceLanguage(flags.language)
+
+    const answer: any = {}
+
+    Object.assign(answer, await prompt([
+      { type: 'input', name: 'name', message: 'What is your project name?', validate: this.questionNameValidate },
+      { type: 'input', name: 'location', message: 'Where do you want to place your project?', default: this.questionLocationDefault },
+      { type: 'confirm', name: 'overwrite', message: 'The folder already exist. Do you want to overwrite?', default: flags.overwrite ?? false, when: this.questionOverwriteWhen, askAnswered: true }
+    ], {
+      name: args.name,
+      location: flags.location,
+      overwrite: flags.overwrite
+    }))
+
+    if (this.shouldOverwrite) this.questionOverwriteValidate(answer.overwrite)
+
+    Object.assign(answer, await prompt([
```

Comment:
having prettier somewhere might be helpful too

Reply:
`standard` or `eslint` already does it's job.